Firestore/Example/Tests/Util/FSTHelpers.mm

```diff
@@ -278,15 +281,14 @@ NSComparator FSTTestDocComparator(const absl::string_view fieldPath) {
 FSTTransformMutation *FSTTestTransformMutation(NSString *path,
                                                NSArray<NSString *> *serverTimestampFields) {
   FSTDocumentKey *key = [FSTDocumentKey keyWithPath:testutil::Resource(util::MakeStringView(path))];
-  NSMutableArray<FSTFieldTransform *> *fieldTransforms = [NSMutableArray array];
+  std::vector<FieldTransform> fieldTransforms{};
+  fieldTransforms.reserve(serverTimestampFields.count);
```

Comment:
Nit: unless the number is expected to be pretty large, this is probably overkill.